dist/npm/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@neondatabase/serverless",
-  "version": "0.7.2",
+  "version": "0.7.3",
```

Comment:
Possibly this should be a minor rather than a patch level change, since it could plausibly break things for people?

Reply:
I think, that with your suggestions it shouldn't break anything or am I missing something?

Reply:
It's a pretty significant change, but I suggest to keep it as patch, to ensure that all who use the package would get the update at least on package-lock regeneration, and by dependabot pick up

Reply:
I also expect that there are no breaking changes in the pr (didn't look through all of it yet)

Reply:
yea, we can break stuff. Let's do a minor version?

Reply:
Done